src/stores/FileBrowserStore.ts

```diff
@@ -367,24 +423,40 @@ export class FileBrowserStore {
         }
 
         const dataType = CARTA.ColumnType.String;
-        columnsData.set(0, {dataType, data: nameData});
-        columnsData.set(1, {dataType, data: unitData});
-        columnsData.set(2, {dataType, data: typeData});
-        columnsData.set(3, {dataType, data: descriptionData});
+        columnsData.set(0, { dataType, data: nameData });
+        columnsData.set(1, { dataType, data: unitData });
+        columnsData.set(2, { dataType, data: typeData });
+        columnsData.set(3, { dataType, data: descriptionData });
 
         let columnHeaders = [
-            new CARTA.CatalogHeader({name: "Name", dataType, columnIndex: 0}),
-            new CARTA.CatalogHeader({name: "Unit", dataType, columnIndex: 1}),
-            new CARTA.CatalogHeader({name: "Data Type", dataType, columnIndex: 2}),
-            new CARTA.CatalogHeader({name: "Description", dataType, columnIndex: 3})
+            new CARTA.CatalogHeader({ name: "Name", dataType, columnIndex: 0 }),
+            new CARTA.CatalogHeader({ name: "Unit", dataType, columnIndex: 1 }),
+            new CARTA.CatalogHeader({ name: "Data Type", dataType, columnIndex: 2 }),
+            new CARTA.CatalogHeader({ name: "Description", dataType, columnIndex: 3 })
         ];
 
-        return {columnHeaders: columnHeaders, columnsData: columnsData};
+        return { columnHeaders: columnHeaders, columnsData: columnsData };
     }
 
-    constructor() {
-        makeObservable(this);
-        this.exportCoordinateType = CARTA.CoordinateType.WORLD;
-        this.exportFileType = CARTA.FileType.CRTF;
+    /// Transfer the request from stokes string
+    /// to [start, end, stride]
+    @computed get saveStokesRange(): number[] {
+        // [start, end, stride] for "ABCD"
```

Comment:
if it's stokes, we should use IQUV instead of ABCD

Reply:
It's only a semantic demo to quick explain the orders. Will use "IQUV" insteadly.